Dockerfile

```diff
@@ -17,6 +17,8 @@ RUN pip install mysql-connector mysqlclient sqlalchemy psycopg2-binary
 
 RUN pip install canvasapi pandas
 
+RUN pip install google-cloud-bigquery
+
 RUN pip install PyMySQL
```

Comment:
All the install should be happening from `requirments.txt` and we should only be doing all the installs with one command `RUN pip install -r requirements.txt`

Reply:
fixed!